### PR TITLE
feat: share template

### PR DESCRIPTION
**Pull request by drdelambre**
we should be able to clone a notebook by it's share id. this PR aims to do that. not expecting to drive traffic here until ephemeral notebooks is done and we have a clearer picture on how this might work

### CODE REVIEW

File: src/flows/templates/types/share.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default register =>
+  register({
+    type: 'share',
+    init: accessID =>
+      fetch(`/api/share/${accessID}`)
```
Comment (genehynson) [inline src/flows/templates/types/share.ts:5]: this is very slick. Nicely done

File: src/flows/templates/types/share.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        .then(res => res.json())
+        .then(res => {
+          delete res.id
```
Comment (genehynson) [inline src/flows/templates/types/share.ts:8]: anything else we would need to delete? What about the orgID?

**Review by genehynson — APPROVED** [review]
took me a second to figure out how this works. For anyone else checking out this PR, go to https://[ns].a.influxcloud.dev.local/notebook/from/share/[accessID]

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2677)

**Comment by genehynson** [comment]
@drdelambre maybe 3rd time's the charm? 😆

**Comment by drdelambre** [comment]
Oh I sure hope so, debugging the prod assets is.... Not fun

**Comment by drdelambre** [comment]
oh wow! just keep merging ™️

File: src/flows/templates/types/share.ts (hunk 1 above)
Comment (asalem1) [reply src/flows/templates/types/share.ts:5]: Don't we have a generated function we can use here?

**Pull request merged**